test_app/CMakeLists.txt

```diff
@@ -7,6 +7,7 @@ set(EXTRA_COMPONENT_DIRS "../components")
 include($ENV{IDF_PATH}/tools/cmake/version.cmake) # $ENV{IDF_VERSION} was added after v4.3...
 if("${IDF_VERSION_MAJOR}.${IDF_VERSION_MINOR}" VERSION_LESS "4.4")
     set(EXCLUDE_COMPONENTS "es8311")
+    set(EXCLUDE_COMPONENTS "esp_lcd_touch")
```

Comment:
You are resetting the `EXCLUDE_COMPONENTS` here. Either you can set the variable in one command
```
set(EXCLUDE_COMPONENTS "esp_lcd_touch" "es8311")
```

Or you list append like this:
```
list(APPEND EXCLUDE_COMPONENTS "esp_lcd_touch")
```

Reply:
Thanks!